src/agent/sqlite_manager/tests/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+find_package(GTest REQUIRED)
+
+add_executable(test_SQLiteManager sqlite_manager_test.cpp)
+target_include_directories(test_SQLiteManager PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../include ${CMAKE_CURRENT_SOURCE_DIR}/../src)
+target_link_libraries(test_SQLiteManager PUBLIC
```

Comment:
Looks like these dependencies can be `PRIVATE`.

Reply:
Done